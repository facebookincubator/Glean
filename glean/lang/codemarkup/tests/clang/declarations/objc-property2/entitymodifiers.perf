{ "@generated": null, "cxx1.ObjcPropertyDeclaration.5": 1 }