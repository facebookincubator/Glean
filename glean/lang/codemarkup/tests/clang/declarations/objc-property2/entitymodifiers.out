[
  "@generated",
  {
    "key": {
      "entity": {
        "cxx": {
          "decl": {
            "objcProperty": {
              "key": {
                "name": { "key": "i" },
                "container": { "interface_": { "key": "I" } },
                "type": { "key": "int" },
                "isInstance": true,
                "isOptional": false,
                "isReadOnly": true,
                "isAtomic": true,
                "source": {
                  "file": { "key": "test.m" },
                  "lineBegin": 11,
                  "columnBegin": 1,
                  "lineEnd": 11,
                  "columnEnd": 26
                }
              }
            }
          }
        }
      },
      "modifiers": {
        "isAbstract": false,
        "isFinal": false,
        "isAsync": false,
        "isStatic": false,
        "isReadonly": true,
        "isConst": false,
        "isMutable": false,
        "isVolatile": false,
        "isVirtual": false,
        "isInline": false
      }
    }
  }
]